railties/lib/rails/api/generator.rb

```diff
@@ -0,0 +1,29 @@
+require "sdoc"
+
+class RDoc::Generator::API < RDoc::Generator::SDoc # :nodoc:
+  RDoc::RDoc.add_generator self
+
+  PUBLIC_CONSTANTS = %w( AbstractController ActionCable ActionController
+                         ActionDispatch ActionMailer ActionView ActiveJob
+                         ActiveModel ActiveRecord ActiveSupport Mime
+                         LoggerSilence Rails SourceAnnotationExtractor )
```

Comment:
I'm not excited about needing to keep this list up to date, nor with the consequences of failing to do so.

How does it look if we list the core classes instead?